diff --git a/node_modules/@snapshot-labs/checkpoint/dist/src/container.js b/node_modules/@snapshot-labs/checkpoint/dist/src/container.js
index 147d7ee..f0035d6 100644
--- a/node_modules/@snapshot-labs/checkpoint/dist/src/container.js
+++ b/node_modules/@snapshot-labs/checkpoint/dist/src/container.js
@@ -8,10 +8,11 @@ const helpers_1 = require("./utils/helpers");
 const register_1 = require("./register");
 const database_1 = require("./utils/database");
 const SCHEMA_VERSION = 1;
-const BLOCK_PRELOAD_START_RANGE = 1000;
+const BLOCK_PRELOAD_START_RANGE = 5000;
 const BLOCK_RELOAD_MIN_RANGE = 10;
 const BLOCK_PRELOAD_STEP = 100;
-const BLOCK_PRELOAD_TARGET = 10;
+const BLOCK_PRELOAD_TARGET = 10000;
+console.log('Block preload target:', BLOCK_PRELOAD_TARGET);
 const BLOCK_PRELOAD_OFFSET = 50;
 const CHECK_LATEST_BLOCK_INTERVAL = 50;
 const DEFAULT_FETCH_INTERVAL = 2000;
